packages/ComponentLibrary/src/components/SecondaryTabs/index.tsx

```diff
@@ -61,8 +73,7 @@ const SecondaryTabs: React.FC<any> = ({ tabsContent, selectedTab = 0, onChange }
           value={selectedTab}
           onChange={handleChange}
           variant="scrollable"
-          scrollButtons="auto"
-        >
+          scrollButtons="auto">
           {tabsContent.slice(0, visibleCount).map((tab: any, index: number) => (
             <Tab
               key={index}
```

Comment:
do not use index as key, instead use some identifier from the **tab** record.
If you don't have a unique identifier inside **tab** object, try creating an ID using something like:
tab.someProp + index.toString()